pkg/artifacts/signable_test.go

```diff
@@ -412,6 +418,48 @@ func TestExtractStructuredTargetFromResults(t *testing.T) {
 							"digest": digest3,
 						}),
 					},
+					{
+						Name: "img2_input" + "_" + ArtifactsInputsResultName,
```

Comment:
nit: I'd include the `sha#` in the name to distinguish these.

Reply:
Done.